src/partials/resource-list/style.scss

```diff
@@ -40,12 +40,9 @@
 
 ul {
   padding: 0;
-  -moz-column-count: 1;
   column-count: 1;
```

Comment:
`column-count` is not working on FF and Chrome because of vendor prefixing. http://caniuse.com/#search=column-count